### ISSUE TITLE
HttpServerConnection [...] empty in -2147483648 days

### ISSUE THREAD

**Issue by ekeih** [Icinga/icinga2#6537]
Hi,

currently we have a 2.9.1 and a 2.8.4 cluster. Both clusters log the following line quite often:

```
[2018-08-09 04:36:07 +0200] information/WorkQueue: #20 (HttpServerConnection) items: 2, rate:  0/s (0/min 0/5min 0/15min); empty in -2147483648 days

# grep -c 'empty in -2147483648 days' /var/log/icinga2/icinga2.log
3229
```

We do not know if the following line is related, but the 2.8.4 cluster also logs:

```
[2018-08-09 03:39:52 +0200] information/HttpServerConnection: Unable to disconnect Http client, I/O thread busy
```

The latter line disappeared with the update to 2.9.1, but the `empty in -2147483648 days` line is still there.

Let us know if there is anything we could do to help to find the cause or the solution.
Thank you in advance!

## Expected Behavior
Instead of `empty in -2147483648 days` the log should contain a valid time in the future.

## Current Behavior
As far as we know it has no negative effects. It seems that only the log is affected.

## Possible Solution
No idea.

## Steps to Reproduce (for bugs)
Unfortunately we have also no idea how to reproduce this. It just happens constantly and we do not know why.

## Your Environment
* Version used (`icinga2 --version`): 2.8.4 and 2.9.1 (see above)
* Operating System and version: Ubuntu 16.04.5 LTS
* Enabled features (`icinga2 feature list`):
```
Disabled features:
Enabled features: api checker command ido-mysql mainlog notification
```

(Accidentally opened https://github.com/Icinga/icingaweb2/issues/3554 first...)

**Comment by dnsmichi** [comment]
This is fixed in #6430.

**Comment by ekeih** [comment]
Perfect! Thank you :)
I searched for related issues but I did not find this PR.

**Comment by ekeih** [comment]
For the record:
In 2.10.0 the change in #6430 fixed the message. We still had a lot of them in our logs without any negative impact.
With 2.10.2 the message disappeared from our logs. My guess is that the TLS fixes in 2.10.2 improved this, but I do not really know ;)
